Framework/LiveData/src/ISIS/ISISHistoDataListener.cpp

```diff
@@ -196,16 +208,9 @@ boost::shared_ptr<Workspace> ISISHistoDataListener::extractData() {
 
   // Create the 2D workspace for the output
   auto localWorkspace = WorkspaceFactory::Instance().create(
-      "Workspace2D", numberOfHistograms, m_numberOfBins[m_timeRegime] + 1,
+      bufferWorkspace, numberOfHistograms, m_numberOfBins[m_timeRegime] + 1,
       m_numberOfBins[m_timeRegime]);
```

Comment:
If you setup the workspace correctly above, you can simplify this and simply use `auto localWorkspace = bufferWorkspace->clone()`, I think.

Reply:
'bufferWorkspace' can't be set up correctly in the 'connect' function, so we need to do like it is done above.